packages/design-system/src/components/disclosure/index.js

```diff
@@ -39,25 +40,29 @@ const rotate = {
  *
  * @param {Object} [props] All props.
  * @param {string} [props.direction] Initial direction of chevron icon
+ * @param {boolean} [props.disabled] Should the icon have a 'disabled' appearance?
```

Comment:
I'm not sure whether we need to expand for this functionality for now, unless you're actively seeing the `disabled` or `aria-disabled` prop in any of the components you've updated here. So I think this could likely be removed for now.

Reply:
I did see it elsewhere, that's what prompted the addition here. It was added just for appearance, but if kept could also prevent rotation of the icon. I'd assume a 'disabled' menu or button would _likely_ not set `isOpen` to `true`?

Reply:
Gotcha! Then I think it makes sense to have it, awesome. Thanks.